### PR TITLE
fix(indicators-modal): update selector

### PR DESCRIPTION
**Pull request by pegaltier**
Fix this issue: https://github.com/akumidv/tradingview-assistant-chrome-extension/issues/183

@akumidv can you merge it?

### CODE REVIEW

**Comment by scheerd** [comment]
Works for Chrome browser, but doesn't seem to work for Brave browser.  Can anyone confirm?

**Comment by scheerd** [comment]
Works for me in Chrome and Brave.  Thanks!

File: content_scripts/selector.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const SEL = {
     strategyReportHeader: '#bottom-area div.backtesting div[class^="reportContainer"] table thead > tr > th',
     strategyReportRow: '#bottom-area div.backtesting div[class^="reportContainer"] table tbody > tr',
     strategyListOptions: 'div[role="listbox"] div[data-name="menu-inner"] div[role="option"] span[class^="label-"]',
-    strategyDefaultElement: '#study-defaults-manager',
+    strategyDefaultElement: 'div[data-name="indicator-properties-dialog"] div[class^="footer-"] div[class^="buttons-"]',
```
Comment (akumidv) [inline content_scripts/selector.js:34]: why not `#property-actions`?

File: content_scripts/ui.js [hunk 2]
```diff
@@ -37,10 +37,10 @@ ui.checkInjectedElements = () => {
       return
     const importExportEl = document.createElement('div')
     importExportEl.id = 'iondvImportExport'
-    importExportEl.setAttribute('style', 'padding-left: 10px;padding-right: 10px')
+    importExportEl.setAttribute('style', 'padding-left: 10px;padding-right: 10px; padding-top: 8px;')
```
Comment (akumidv) [inline content_scripts/ui.js:40]: manually added to current version. Thanks.

File: content_scripts/selector.js (hunk 1 above)
Comment (pegaltier) [reply content_scripts/selector.js:34]: i didn't see it, i agree with this id instead of classes

File: content_scripts/ui.js (hunk 2 above)
Comment (pegaltier) [reply content_scripts/ui.js:40]: perfect then I close this PR

**Pull request closed without merging**